test/fork/dao.test.js

```diff
@@ -357,4 +373,199 @@ describe('Integration tests on Gnosis deployment', () => {
+        ['call(uint256,address,bytes,bytes4)'](10, attacker.address, '0xc0ffee', PNETWORK_NETWORK_IDS.gnosisMainnet)
+    )
+      .to.be.revertedWithCustomError(forwarderHostDandelion, 'InvalidCaller')
+      .withArgs(attacker.address, DANDELION_VOTING_ADDRESS)
+  })
+
+  it('should be possible to pegin to finance vault', async () => {
+    await mintPntOnGnosis(FINANCE_VAULT, '10', '0xc0ffee')
+  })
+})
+
+describe('Integration tests on Ethereum deployment', () => {
+  let vault, forwarderNativePermissioned, pnetwork, faucet, daoVotingV1, tokenHolders, association, ethPnt
+
+  const TOKEN_HOLDERS_ADDRESSES = [
+    '0x100a70b9e50e91367d571332E76cFa70e9307059',
+    '0xc4442915B1FB44972eE4D8404cE05a8D2A1248dA',
+    '0xF03f2303cC57bC5Cd63255749e86Ed8886Ca68Fc',
+    '0xe0EDF3bAee2eE71903FbD43D93ce54420e5933F2'
+  ]
+
+  const missingSteps = async () => {
+    const ForwarderNativePermissioned = await hre.ethers.getContractFactory('ForwarderNativePermissioned')
+    forwarderNativePermissioned = await ForwarderNativePermissioned.deploy(PNT_ON_ETH_ADDRESS, ERC20_VAULT)
+    await forwarderNativePermissioned.whitelistOriginAddress(ZERO_ADDRESS)
+    daoVotingV1 = await hre.ethers.getContractAt(DandelionVotingAbi, DANDELION_VOTING_V1_ADDRESS)
+    // open vote to change inflationOwner
+    const executionScript = encodeCallScript(
+      [
+        [
+          ETHPNT_ADDRESS,
+          ethPnt.interface.encodeFunctionData('whitelistInflationRecipient', [
+            await forwarderNativePermissioned.getAddress()
+          ])
+        ],
+        [
+          ETHPNT_ADDRESS,
+          ethPnt.interface.encodeFunctionData('setInflationOwner', [await forwarderNativePermissioned.getAddress()])
+        ]
+      ].map((_args) => encodeFunctionCall(..._args))
+    )
+    const voteId = await openNewVoteAndReachQuorum(
+      daoVotingV1,
+      association,
+      tokenHolders,
+      executionScript,
+      'change inflation owner?'
+    )
+    expect(await ethPnt.inflationRecipientsWhitelist(await forwarderNativePermissioned.getAddress())).to.be.false
+    await expect(daoVotingV1.executeVote(voteId))
+      .to.emit(daoVotingV1, 'ExecuteVote')
+      .withArgs(voteId)
+      .and.to.emit(ethPnt, 'InflationRecipientWhitelisted')
+      .and.to.emit(ethPnt, 'NewInflationOwner')
+    expect(await ethPnt.inflationRecipientsWhitelist(await forwarderNativePermissioned.getAddress())).to.be.true
+    expect(await ethPnt.inflationOwner()).to.be.eq(await forwarderNativePermissioned.getAddress())
+  }
+
+  beforeEach(async () => {
+    const rpc = hre.config.networks.mainnet.url
+    await hardhatReset(hre.network.provider, rpc)
+    ;[faucet] = await hre.ethers.getSigners()
+    tokenHolders = await Promise.all(TOKEN_HOLDERS_ADDRESSES.map(hre.ethers.getImpersonatedSigner))
+    pnetwork = await hre.ethers.getImpersonatedSigner(PNETWORK_ADDRESS)
+    association = await hre.ethers.getImpersonatedSigner('0xf1f6568a76559d85cF68E6597fA587544184dD46')
```

Comment:
In case we have a different one for gnosis
```suggestion
    associationEthAddress = await hre.ethers.getImpersonatedSigner('0xf1f6568a76559d85cF68E6597fA587544184dD46')
```

Reply:
Note this is an integration test for Ethereum